src/Nancy.Demo.Authentication.Forms.TestingDemo/LoginFixture.cs

```diff
@@ -17,10 +17,10 @@ public LoginFixture()
         }
 
         [Fact]
-        public void Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
+        public async Task Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
         {
             // Given, When
-            var response = browser.Post("/login/", (with) =>
+            var response = await browser.Post("/login/", (with) =>

```

Comment:
I would rather go with a new `SyncBrowser` or `LegacyBrowser` type for the sync versions and have `Browser` be fully async. I don't like introducing the `*Async` methods - once the sync ones are removed we'll want to remove the `Async` suffixes, but that'll be another round of breaking changes.